run-tests.sh

```diff
@@ -3,9 +3,28 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# do nothing on Travis except for the master branch
+TRAVIS=${TRAVIS:-false}
+if [[ "$TRAVIS" == "true" && "$TRAVIS_BRANCH" != "master" ]]; then
```

Comment:
Can't you use .travis.yml's `branches` option?

Reply:
https://github.com/whatwg/misc-server/pull/21